internal/service/health_test.go

```diff
@@ -0,0 +1,44 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	pb "github.com/project-kessel/relations-api/api/health/v1"
+	"github.com/project-kessel/relations-api/internal/biz"
+	"github.com/project-kessel/relations-api/internal/data"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHealthService_GetLivez(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.TODO()
+	spicedb, err := container.CreateSpiceDbRepository()
+	assert.NoError(t, err)
+
+	service := createHealthService(spicedb)
+	resp, err := service.GetLivez(ctx, &pb.GetLivezRequest{})
+
+	assert.NoError(t, err)
+	assert.Equal(t, resp, &pb.GetLivezReply{Status: "OK", Code: 200})
+}
+
+func TestHealthService_GetReadyz_SpiceDBAvailable(t *testing.T) {
```

Comment:
For a negative test here, you could create the SpiceDB repository with an invalid config or use a test double (a different struct that implements ZanzibarRepository and returns false from IsBackendAvailable

Reply:
Utilized a test double to just return an error and test the response.